src/pods/toolbar/components/export-button/export-button.business.ts

```diff
@@ -28,6 +28,78 @@ export const calculateTableEndYPosition = (table: TableVm): number => {
   return table.y + tableHeight;
 };
 
+export const isTableOverlap = (table: TableVm, tables: TableVm[]): boolean => {
+  if (tables.length === 0) {
+    return false;
+  }
+
+  const expandedFields = expandTableFields(table.fields);
+  const tableEndXPosition = table.x + TABLE_CONST.TABLE_WIDTH;
+  const tableEndYPosition = calculateTableEndYPosition({
+    ...table,
+    fields: expandedFields,
+  });
+
+  return tables.some(t => {
```

Comment:
instead of _t_ use something more meaningful in this case you are going to use it everywhere, we could call it something like:

tableA
tableB

By doing this we now that tableA is the original table and table B the one we are iteraring currenly

Reply:
done